tap_exacttarget/endpoints/data_extensions.py

```diff
@@ -97,16 +111,22 @@ def _get_fields(self, extensions):
             self.auth_stub)
 
         for field in result:
+            is_replication_key = False
```

Comment:
@harshpatel4crest Can you please write the logic for this loop, explaining the motive behind initializing these variables

Reply:
Added a comment describing the working of the loop